tests/test_utils.py

```diff
@@ -98,10 +98,10 @@ def test_deobfuscate_email(self):
 
     def test_html_to_text(self):
         html_link = u"<strong><a href='https://hasjob.co'>Hasjob</a></strong>"
-        unsafe_link = u"<strong><a href='https://example.com?x=1&amp;y=bangalore '>Hasjob</a></strong>"
+        unsafe_link = u"<strong><a href='https://hasjob.co/?l=bangalore&c=programming '>Hasjob</a></strong>"
         markdown_link = u'**[Hasjob](https://hasjob.co)**\n\n'
         formatted_link = u'**Hasjob - https://hasjob.co**\n\n'
-        encoded_link = u'**Hasjob - https://example.com?x=1&y=bangalore%20**\n\n'
+        encoded_link = u'**Hasjob - https://hasjob.co/?l=bangalore&amp;c=programming%20**\n\n'
```

Comment:
The encoded version should not have `&amp;` in plain text output. It should be `&`. This is actually a bug in the library.